native/packages/react-native-bpk-component-touchable-native-feedback/src/BpkTouchableNativeFeedback.js

```diff
@@ -50,7 +50,7 @@ const BpkTouchableNativeFeedback = (props: Props) => {
 
 BpkTouchableNativeFeedback.propTypes = {
   children: PropTypes.element.isRequired,
-  borderlessBackground: PropTypes.boolean,
+  borderlessBackground: PropTypes.bool,
```

Comment:
I keep seeing an error in storybooks to do with this - I expect this line will fix it! Nice one 👍